go-app-ussd_popi_rapidpro.js

```diff
@@ -44,6 +44,53 @@ go.Hub = function() {
     return Hub;
 }();
 
+go.Engage = function() {
```

Comment:
We shouldn't have this block in here at all, maybe do a pull to update your gruntfile